Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
I think either solution would be ok: allow Vagrant to use current versions of CoreOS (knowing that Docker 1.6.2 may be buggy), or make the stable channel code conditional in the Vagrantfile if possible. Or we can just disable the test-coreos-beta and test-coreos-alpha jobs in CI temporarily, until this overall situation is improved--they are valuable, but not crucial.